pyuvdata/uvdata/mir_parser.py

```diff
@@ -47,6 +72,12 @@ class MirParser(object):
     def __init__(
         self,
         filepath=None,
+        *,
+        auto_defaults=None,
+        compass_soln=None,
+        make_v3_compliant=None,
```

Comment:
```suggestion
        make_v3_compliant=False,
```
should be a bool?

Reply:
See comment below -- using the None to allow for dynamic default behavior depending on file version.